### PR TITLE
Small design tweaks

### PR DESCRIPTION
**Pull request by LukasHirt**
Change color design tokens, unite spacing and create custom util classes for them, enable `v-model` in checkbox and radio inputs and fix search bar icons.

### CODE REVIEW

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

File: src/styles/theme/oc-spinner.scss [hunk 1]
```diff
@@ -32,32 +30,37 @@ $oc-spinner-size: 24px;
     position: relative;
   }
 
-  &,
-  &-small {
-    @include oc-spinner-size(1);
+  @include oc-spinner-size(1);
```
Comment (micbar) [inline src/styles/theme/oc-spinner.scss:33]: Codacy found an issue: [Expected blockless @include to come before rule (order/order)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

File: src/styles/theme/oc-spinner.scss (hunk 1 above)
Comment (micbar) [inline src/styles/theme/oc-spinner.scss:33]: Codacy found an issue: [Expected blockless @include to come before rule (order/order)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

File: src/elements/OcSidebarNavItem.vue [hunk 2]
```diff
@@ -53,7 +53,7 @@ export default {
         classes.push("uk-active")
       }
       if (this.isolate) {
-        classes.push("uk-margin-medium-top")
+        classes.push("oc-mt-m")
```
Comment (kulmann) [inline src/elements/OcSidebarNavItem.vue:56]: Can be `oc-mt` because that's already medium

File: src/styles/theme/oc-alert.scss [hunk 3]
```diff
@@ -46,7 +46,7 @@
 }
 
 .oc-alerts-transition-leave-to {
-  transform: translateY(-$gutter);
+  transform: translateY(-space-medium);
```
Comment (kulmann) [inline src/styles/theme/oc-alert.scss:49]: misses the `$` sign for the variable

File: src/styles/theme/oc-app-menu.scss [hunk 4]
```diff
@@ -2,7 +2,7 @@
   @extend .uk-nav;
   @extend .uk-nav-default;
   @extend .uk-nav-parent-icon;
-  @extend .uk-margin-medium-top;
+  @extend .oc-mt-m;
```
Comment (kulmann) [inline src/styles/theme/oc-app-menu.scss:5]: Can be `oc-mt` (already medium by default)

**Review by kulmann — CHANGES_REQUESTED** [review]

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Review by kulmann — APPROVED** [review]
🚀

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

File: src/styles/theme/oc-checkbox.scss [hunk 5]
```diff
@@ -3,10 +3,68 @@
...
+}
+
 .oc-checkbox {
-  @extend .uk-checkbox;
+  @include oc-form-check-size(1);
+
+  display: inline-block;
+  overflow: hidden;
+  vertical-align: middle;
+  -webkit-appearance: none;
```
Comment (micbar) [inline src/styles/theme/oc-checkbox.scss:17]: Codacy found an issue: [Expected -webkit-appearance to come before vertical-align (order/properties-alphabetical-order)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

File: src/styles/theme/oc-checkbox.scss (hunk 5 above)
Comment (micbar) [inline src/styles/theme/oc-checkbox.scss:17]: Codacy found an issue: [Expected -webkit-appearance to come before vertical-align (order/properties-alphabetical-order)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

File: src/styles/theme/oc-checkbox.scss (hunk 5 above)
Comment (micbar) [inline src/styles/theme/oc-checkbox.scss:17]: Codacy found an issue: [Expected -webkit-appearance to come before vertical-align (order/properties-alphabetical-order)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/utils/sizeClasses.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785&bid=20370810)

**Pull request merged**